app/views/puppetclasses/_form.html.erb

```diff
@@ -47,7 +47,6 @@
               <li data-used-environments=<%= key.environments.map(&:to_s).to_json %> >
                 <a data-toggle="tab" id="pill_<%= key.to_s.gsub(' ','_') + '_' + key.id.to_s %>" href="#<%= key.to_s.gsub(' ','_') + '_' + key.id.to_s %>" title="<%= key %>">
                   <div class="clip"><%= icon_text((key.override ? "flag": ""), key.to_s.gsub('_',' ')) %></div>
-                  <span class="close pull-right">&times;</span>
```

Comment:
it doesn't, this part is duplicated (line 50 and 77):
https://github.com/theforeman/foreman/blob/develop/app/views/puppetclasses/_form.html.erb#L77
part of this could be refactored to a partial and add an if for the `close` part but I think that's outside the scope of this PR.